doc/content/the-things-stack/host/kubernetes/azure/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Azure"
+description: ""
+weight: 1
+---
+
+{{% tts %}} can be deployed to Azure as a set of highly available services on Azure Kubernetes Service using Terraform and Helm.
+
+This guide gives an overview of the architecture and shows how to deploy your own highly available {{% tts %}} cluster.
```

Comment:
This guide needs an architecture diagram http://localhost:1313/the-things-stack/host/aws/ecs/architecture/